app/keepers/precompiles.go

```diff
@@ -0,0 +1,79 @@
+package keepers
+
+import (
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/codec"
+	precisebankkeeper "github.com/cosmos/evm/x/precisebank/keeper"
+	"maps"
+
+	evidencekeeper "cosmossdk.io/x/evidence/keeper"
+	govkeeper "github.com/cosmos/cosmos-sdk/x/gov/keeper"
+	slashingkeeper "github.com/cosmos/cosmos-sdk/x/slashing/keeper"
+	bankprecompile "github.com/cosmos/evm/precompiles/bank"
+	"github.com/cosmos/evm/precompiles/bech32"
+	evidenceprecompile "github.com/cosmos/evm/precompiles/evidence"
+	govprecompile "github.com/cosmos/evm/precompiles/gov"
+	"github.com/cosmos/evm/precompiles/p256"
+	slashingprecompile "github.com/cosmos/evm/precompiles/slashing"
+	erc20Keeper "github.com/cosmos/evm/x/erc20/keeper"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/vm"
+)
+
+const bech32PrecompileBaseGas = 6_000
+
+// NewAvailableStaticPrecompiles adds the static precompiles to the EVM
+// TODO: Add custom staking wrapper precompile here, IBC precompile and distribution precompile
+func NewAvailableStaticPrecompiles(
+	cdc codec.Codec,
+	bankKeeper precisebankkeeper.Keeper,
+	erc20Keeper erc20Keeper.Keeper,
+	govKeeper govkeeper.Keeper,
+	slashingKeeper slashingkeeper.Keeper,
+	evidenceKeeper evidencekeeper.Keeper,
+) map[common.Address]vm.PrecompiledContract {
+	// TODO: We can add more custom precompiles here for Babylon Modules
+
+	// Clone the mapping from the latest EVM fork.
+	precompiles := maps.Clone(vm.PrecompiledContractsBerlin)
```

Comment:
latest updated precompiles would be `PrecompiledContractsPrague`
And by default, this is the active fork on the evm config